shared_model/validators/field_validator.hpp

```diff
@@ -160,33 +200,39 @@ namespace shared_model {
         iroha::ts64_t now = iroha::time::now();
         // TODO 06/08/17 Muratov: make future gap for passing timestamp, like
         // with old timestamps IR-511 #goodfirstissue
+
         if (now < timestamp) {
-          reason.second.push_back(boost::str(
-              boost::format(
-                  "timestamp broken: send from future (%llu, now %llu)")
-              % timestamp % now));
+          auto message = (boost::format("bad timestamp: sent from future, "
+                                        "timestamp: %llu, now: %llu")
+                          % timestamp % now)
+                             .str();
+          reason.second.push_back(std::move(message));
         }
 
-        if (now - timestamp > MAX_DELAY) {
-          reason.second.push_back(boost::str(
-              boost::format("timestamp broken: too old (%llu, now %llu)")
-              % timestamp % now));
+        if (now - timestamp > max_delay) {
+          auto message =
+              (boost::format(
+                   "bad timestamp: too old, timestamp: %llu, now: %llu")
+               % timestamp % now)
+                  .str();
+          reason.second.push_back(std::move(message));
         }
       }
 
       void validateCounter(ReasonsGroupType &reason,
                            const interface::types::CounterType &counter) const {
-        if (counter == 0) {
-          reason.second.push_back(
-              boost::str(boost::format("Counter should be > 0")));
+        if (counter <= 0) {
+          reason.second.push_back("Counter should be > 0");
         }
       }
 
      private:
-      std::regex account_id_, asset_id_, ip_address_, name_, detail_key_;
+      std::regex account_id_regex, asset_id_regex, name_regex, detail_key_regex;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Variable_Names

>Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.

Reply:
And probably it is better to write all strings in separate lines, like:
```
std::regex account_id_regex_;
std::regex asset_id_regex_;
...
```